app/src/main/java/com/framgia/fsalon/screen/customer/CustomerPresenter.java

```diff
@@ -18,5 +43,43 @@ public void onStart() {
 
     @Override
     public void onStop() {
+        mCompositeDisposable.clear();
+    }
+
+    @Override
+    public void getCustomers(int page, int perpage) {
+        Disposable disposable = mBookingCustomerRepository.getListCustomer(page,perpage)
+            .subscribeOn(Schedulers.io())
+            .observeOn(AndroidSchedulers.mainThread())
+            .doOnSubscribe(new Consumer<Disposable>() {
+                @Override
+                public void accept(@NonNull Disposable disposable) throws Exception {
+                mViewModel.showLoadMore();
```

Comment:
ERROR: 'method def' child have incorrect indentation level 16, expected level should be one of the following: 20, 24.